src/rules/exports-last.js

```diff
@@ -0,0 +1,27 @@
+const isExportStatement = ({ type }) =>
+  type === 'ExportDefaultDeclaration'
+  || type === 'ExportNamedDeclaration'
+  || type === 'ExportAllDeclaration'
+
+const rule = {
+  create(context) {
+    return {
+      Program({ body }) {
+        const firstExportStatementIndex = body.findIndex(isExportStatement)
+
+        if (firstExportStatementIndex !== -1) {
+          body.slice(firstExportStatementIndex).forEach((node) => {
+            if (!isExportStatement(node)) {
+              context.report({
+                node,
+                message: 'Export statements should appear at the end of the file',
+              })
+            }
+          })
+        }
+      },
+    }
+  },
+}
+
+export default rule
```

Comment:
Would you mind making the file structure (exports and stuff) look a bit more like the others, for consistency? Here's an example: https://github.com/benmosher/eslint-plugin-import/blob/master/src/rules/no-dynamic-require.js